packages/server/modules/core/tests/generic.spec.js

```diff
@@ -157,25 +157,23 @@ describe('Generic AuthN & AuthZ controller tests', () => {
       envHelperMock.resetMockedFunctions()
     })
     it('should allow stream:owners to be stream:owners', async () => {
-      const role = await authorizeResolver(
+      await authorizeResolver(
```

Comment:
Nothing outside of tests uses the return, and the return was very dynamic before - it could return server roles, stream roles, even fake roles if admin override was enabled. I decided to just scrap the otherwise `any` return, since nothing really needs it.